src/components/graphQLConnect.ts

```diff
@@ -0,0 +1,59 @@
+import fetch from 'node-fetch'
+import * as config from '../config.json'
+
+const endpoint = config.graphQLendpoint
+
+class GraphQLdb {
```

Comment:
I can see some use cases where GraphQL could be simpler when retrieving deep child relationships. 

not sure if this particular use case warrants a GraphQL statement, but I guess good start. Although I think would be nicer to review two PR's one for sequential actions and one for introducing GraphQL. 
No changes required just a minor comment

Reply:
Yeah, I only started making the GraphQL abstraction when I was writing the `getTemplateId` method and realised that the query to get the original template id from some of the deeper tables (e.g. `review_response`) would be much easier to write with GraphQL.

However, I haven't actually implemented those queries yet, as we don't have Triggers on the tables nor any mock data to work with, so left them as To-Dos for now. (Only `application` and `review` table triggers have `getTemplateId` implemented so far, in SQL.)